packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -660,7 +656,8 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
   }
 
   private _onColumnResized(resizingColumn: IColumn, newWidth: number) {
-    let newCalculatedWidth = Math.max(resizingColumn.minWidth || MIN_COLUMN_WIDTH, newWidth);
+    const minWidth = Math.min(resizingColumn.minWidth || MIN_COLUMN_WIDTH, MIN_COLUMN_WIDTH);
+    let newCalculatedWidth = Math.max(minWidth, newWidth);
```

Comment:
nit, `const`?

Reply:
Is `const` not preferable to `let` for variables that don't change after initialized?

Reply:
yeah, so we should use it :)

Reply:
Oh... you're pointing out the existing `let` on declaring newCalculatedWidth.  I wasn't planning to arbitrarily change existing code.

Reply:
You touched the line anyway! But yeah was a nit to begin with :)